test/regress/regress0/seq/len_simplify_unknown.smt2

```diff
@@ -0,0 +1,5 @@
+; COMMAND-LINE: -q
```

Comment:
I would omit this benchmark.  It isn't very useful to test that something doesn't work, since this will only lead to changing if e.g. the option you give becomes default or another technique solves this by default.